code/Test_definitions/Geofencing.feature

```diff
@@ -0,0 +1,179 @@
+@Geofencing 
+Feature: Camara Geofencing Subscriptions API ,v0.3.0 - Operations on subscriptions 
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in geofencing-subscriptions.yaml, version v0.3.0
+
+  Background: Common Geofencing Subscriptions setup
+    Given the resource "{apiroot}/geofencing-subscriptions/v0.3/"  as geofencing base-url                                                           
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+
+  @geofencing_subscriptions_01_Create_geofencing_subscription_for_a_device
+  Scenario:  Create geofencing subscription
+    Given a valid subscription request body 
+    When the  request "createSubscription" is sent 
+    Then the response code is 201
+    And the response header "Content-Type" is "application/json"
+    And the response header "x-correlator" has same value as the request header "x-correlator"
+    And the response body complies with the OAS schema at "/components/schemas/Subscription"
+
+  @geofencing_subscriptions_02_Operation_to_retrieve_list_of_subscriptions
+  Scenario: Get a list of subscriptions.
+    Given the request body is not available
```

Comment:
@jlurien I'm not sure what that means.. The case is here, that instead of a specific request body here we explicitly shouldn't have a request body.

Reply:
@jlurien  IMO As we are keeping request body everywhere for creation of subscription then it makes sense to showcase no request body required for get list ,get & delete method .

Reply:
I see, I thought it's an equivalent to the empty body :-) 
So basically a precondition, that already some subscriptions exist.
We could make 2 cases of it, one to return an empty list and one with subscriptions. Currently the evaluation is generic with "if any", but it could be more specific then.

Reply:
Yes, we split can this into 2 scenarios one with 

Given a client with subscriptions created
response is an array of items complying with the schema

other

Given a client without subscriptions created
response is an empty array []